<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeployHelper</class>
 <widget class="QMainWindow" name="DeployHelper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1764</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1640</width>
    <height>760</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#364052;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: &quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
color:white;</string>
    </property>
    <property name="text">
     <string>Your Deploys:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_AddDeploy">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    /*border: 2px solid #000000;*/
    border-radius: 8px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #2c509e, stop: 1 #0092d1);
    min-width: 80px;
	color:white;
	font-family: &quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
	font-size: 15px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #0092d1, stop: 1 #2c509e);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
    </property>
    <property name="text">
     <string>Add Deploy</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>1331</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
   /* border: 2px solid #0092d1;*/
	 border: 2px solid white;
    border-radius: 5px;
    padding: 0 8px;
    selection-background-color: darkgray;
	font-family: &quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
	color:white;
font-size: 15px;
}</string>
    </property>
    <property name="title">
     <string>Manage Deploy</string>
    </property>
    <widget class="QPushButton" name="Btn_StartDeploy">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>620</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    /*border: 2px solid #000000;*/
    border-radius: 8px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #2c509e, stop: 1 #0092d1);
    min-width: 80px;
	color:white;
	font-family: &quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
	font-size: 15px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #0092d1, stop: 1 #2c509e);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
     </property>
     <property name="text">
      <string>Start Deploy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: &quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
color:white;</string>
     </property>
     <property name="text">
      <string>Deploy Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_DeployName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*border: 1px solid #0092d1;*/
border: 0px solid #0092d1;
    border-radius: 5px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;
	font-family: &quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
font-size: 14px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: &quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
color:white;</string>
     </property>
     <property name="text">
      <string>Source Files:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_SourceFiles">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>591</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
   /* border: 1px solid #0092d1;*/
	border: 0px solid #0092d1;
    border-radius: 5px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;
	font-family: &quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
	font-size: 14px;
}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="placeholderText">
      <string>Source Files</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>90</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: &quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
color:white;</string>
     </property>
     <property name="text">
      <string>Destination Path:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_DestPath">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>130</y>
       <width>651</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    /*border: 1px solid #0092d1;*/
border: 0px solid #0092d1;
    border-radius: 5px;
    padding: 0 8px;
    background: white;
    selection-background-color: #2c509e;
	font-family: &quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
font-size: 14px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Enter the destination path</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget_Deploys">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>251</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
    /*border: 2px solid #0092d1;*/
 border: 2px solid white;
    border-radius: 5px;
    padding: 0 8px;
    selection-background-color: darkgray;
	font-family: &quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
}


 QListWidget::item:selected {
                                  border : 0px solid black;
                                  /*background : #0092d1;*/
									border-radius: 1px;
									background : #9ca2ad;
										color:white;
									font-family: &quot;Lucida Console&quot;, &quot;Courier New&quot;,monospace;
										
                                  }
 QListWidget::item {
                                  border : 0px solid black;
									font-family: &quot;Lucida Console&quot;, &quot;Courier New&quot;,monospace;
color:white;
                                  }




</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1764</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
